File: AirHockeyServer/AirHockeyServer/Entities/EditionCommand/PortalCommand.cs
using AirHockeyServer.Entities.Messages.Edition;

namespace AirHockeyServer.Entities.EditionCommand
{
    public class PortalCommand : AbstractEditionCommand
    {
        public float[] StartPosition { get; set; }
        public float[] EndPosition { get; set; }

        public PortalCommand(string objectUuid) : base(objectUuid)
        {
        }
    }
}

File: AirHockeyLightClient/AirHockeyLightClient/InterfaceGraphique/Editor/Commands/PortalCommand.swift
///////////////////////////////////////////////////////////////////////////////
/// @file PortalCommand.swift
/// @author Mikael Ferland
/// @date 2017-10-30
/// @version 1
///
/// @addtogroup log3900 LOG3990
/// @{
///////////////////////////////////////////////////////////////////////////////

class PortalCommand: AbstractEditionCommand {
    
    var objectUuid: String = ""
    var endUuid: String = ""
    var startPosition: [Float] = []
    var endPosition: [Float] = []
    
    init(objectUuid: String) {
        self.objectUuid = objectUuid
    }
    
    func executeCommand() {
        FacadeModele.instance.creerNoeuds(type: "", nomType: "")
    }
    
}

///////////////////////////////////////////////////////////////////////////////
/// @}
///////////////////////////////////////////////////////////////////////////////
